book/browsersandweb.md

```diff
@@ -0,0 +1,623 @@
+[WHATWG](https://whatwg.org/), but [CSS](https://drafts.csswg.org/) and other
+features are still standardized at the W3C. JavaScript is standardized at
+[TC39](https://tc39.es/) (“Technical Committee 39” at
+[ECMA](https://www.ecma-international.org/memento/history.htm), yet another
+standards body). [HTTP](https://tools.ietf.org/html/rfc2616) is standardized by
+the [IETF](https://www.ietf.org/about/).
+
+In the early days, it was not so clear that browsers would remain standard and
+that one browser might not end up “winning” and becoming another proprietary
+software platform. There are multiple reasons this didn’t happen, among them
+the egalitarian ethos of the computing community and the presence and strength
+of the W3C.  Equally important was the networked nature of the web, and
+therefore the desire of websites to make sure their site worked correctly in
+most or all of the browsers (otherwise they would lose customers), leading
+them to avoid any proprietary extensions.
+
+Despite fears that this might happen, there never really was a point where any
+browser openly attempted to break away from the standard. Instead, intense
+competition for market share was channeled into very fast innovation and an
+ever-expanding set of APIs and capabilities for the web, which we nowadays
+refer to as _the Web Platform,_ not just the “World Wide Web”. This recognizes
+the fact that the web is no longer a document viewing mechanism, but has
+evolved into a fully realized computing platform and ecosystem[^25]. Given
+these outcomes, it becomes clear that it’s not so relevant to know which
+browser “won” or “lost” each of the browser “wars”. In both cases _the web won_
+and was preserved and enhanced for the benefit of the world. In economics
+terms, enforcing a standard set of APIs across browsers made the web platform a
+_commodity_; instead of competing based on lock-in, browsers compete on
+_performance_, and also _browser features_ that are not part of the web
+platform - for example, tabbed UIs and search engine integration. Browser
+development is also primarily funded by revenue from search engine
+advertisements; a secondary typical funding motivation is to improve the
+competitive position of an operating system or device owned or controlled by
+the company building the browser[^26].
+
+An important important and interesting outcome of the second browser war was
+that all mainstream browsers today (of which there are *many* more than three)
+are based on _three open-source web rendering / JavaScript engines_: Chromium,
+Gecko and WebKit[^27]. Further, all three of these engines date back to the
+1990s - i.e. almost to the beginning of the web, and two of them (Chromium and
+WebKit) have a common ancestral codebase. Gecko is an open-source descendant of
+code from Netscape. That this occurred is not an accident, and in fact tells us
```

Comment:
```suggestion
Gecko and WebKit[^27]. Since Chromium and
WebKit have a common ancestral codebase, while Gecko is an open-source descendant of
Netscape, all three date back to the 1990s---almost to the beginning of the web.
That this occurred is not an accident, and in fact tells us
```